extensions/amp-viewer-integration/amp-doc-viewer-api.md

```diff
@@ -0,0 +1,124 @@
+
+| Parameter             | Data type             | Description                                           |
+|-----------------------| ----------------------|-------------------------------------------------------|
+|`cid`|`boolean`|Whether to request the Base Client ID from the Viewer.|
+|`csi`|`boolean`|Whether the Viewer will collect latency metrics from the Document.|    
+|`dialog`|`boolean`|Whether the Viewer will take responsibility for opening a dialog for an AMP Access login. If false, opening a login dialog does not involve the Viewer. If true, the Viewer must support openDialog.|
+|`development`| `boolean`|Whether the AMP runtime is in development mode. Development mode will validate the AMP document and show its results in the developer console.|
+|`history`|`boolean`| Whether the AMP Viewer will take over history management. If true, the viewer must support popHistory, pushHistory, and historyPopped.|
+|`log`|`boolean`| Whether to turn on logging in the AMP runtime.|
+|`paddingTop`| `integer`|The paddingTop to apply to the document body in pixels. This is used to allow space for a title bar.|
+|`prerenderSize`|`integer`|The number of vertical screens to prerender. Resources within the range may be loaded before the document is visible.|
+|`p2r`|`boolean`| Whether to Pull to Refresh is enabled. pull-to-refresh.js|
+|`referrer`|`string`|The referrer to use for AMP analytics.|
+|`storage`|`boolean`|Whether the Viewer will provide local storage. If true, then the Viewer must support loadStore and saveStore.|
+|`viewerUrl`|`string`|The url of the Viewer. This is the shareable Viewer URL and may be used as a fallback as part of AMP Access.|
+|`webview`|`boolean`|Whether the document is being loaded in a WebView instead of an iframe. This will enable "embedded" mode for the document, which is needed for dialog to work.|
+|`highlight`|`string`|The information to highlight text. JSON-encoded string.|
+|`viewportType`| natural, natural-ios-embed|The viewport type. Natural means that the AMP document can read the window dimensions and they will be correct. natural-ios-embed Viewport mode is for use when the AMP document is displayed in an iframe on Mobile Safari. It works around quirks of Mobile Safari's iframe implementation by making the body scrollable rather than the document.|
+|`visibilityState`|'hidden' (deprecated, use 'inactive'), 'inactive','paused','visible',‘prerender’|The initial visibility state of the AMP document.<ul><li>prerender- AMP document is being pre-rendered before being shown. It may be prerendered according to prerenderSize</li><li>inactive or hidden -The document is not visible to the user.</li><li>paused - this document should stop loading new resources and pause any resources such as video. The document may still be visible to the user, but the viewer may be performing a performance sensitive operation such as a swipe.</li><li>visible- The document is active and visible to the user.</li>|
```

Comment:
are these supposed to be backticks or quotes on the datatypes in this entry? Also, could we move the deprecated value under the valid values?

Reply:
fixed to use backticks. Moved deprecated to last.